---
title: "Manuscript for Cyanobacteria: A novel Species"
subtitle: "Project 3"
author: "Chiyoko Onouye"
date: "`r Sys.Date()`"
format: html
bibliography: Cyano.bib
csl: apa.csl
---

# Abstract

*Cyanobacteria are a diverse group of Gram-negative oxygen-evolving photoautotrophic prokaryotes which play an important role in biological nitrogen fixation. These prokaryotes produce two metabolites: primary metabolites, essential for survival, and secondary metabolites required for auxiliary purposes--stress responses, defense mechanisms, metal carrying, and signaling. These secondary metabolites can include compounds that act as hormones, antibiotics, allelochemicals, and toxins. This is important because antibiotic resistance in bacteria and fungi is becoming a problem due to resistant pathogenic bacteria. Both species of cyanobacteria used in this experiment belong to a Genera with known toxin, antibiotic, and/or fungi-producing species. Thus we aim to expand upon this work by investigating these newly identified species and their production of secondary metabolites. Specifically, we aim to 1) identify both novel cyanobacteria by doing whole genome sequencing and creating phylogenetic trees based on known conserved regions as well as 16s phylogenetic trees, 2) investigate secondary metabolites by annotating the genome sequenced, 3) test the reactivity of the antimicrobial and/or antifungal compounds produced by the cyanobacteria against Gram-Positive bacteria, Gram-Negative bacteria, and a fungal species, and to evaluate the potential toxicity of both species using an Artemia salina bioassay.*

{{< pagebreak >}}

# Introduction

## General Background Information

**1.1 Beginning of life, Cyanobacteria**

Cyanobacteria or Cyanophyta are an ancient group of oxygenic photosynthesizers estimated to have originated around 3.0 ga. The appearance of cyanobacteria led to the Great Oxygenation Event (GOE) approximately 2.4 billion years ago, thus greatly modifying the Earth and its atmospheres[@guguen_2021_origin]. These changes gave rise to new ecosystems due to the difference in climate, the reduction of atmospheric carbon dioxide, and the increased sunlight. Through acquiring the chloroplast and mitochondria via primary endosymbiosis, the rise of eukaryotes can be dated back to approximately \~1,200 million years ago [@shih_2013_primary]. A second oxygenation event can be seen during the Neoproterozoic Oxygenation Event (NOE) approximately 800-600 million years ago due to the rise of algae. 

 Since 2009, there are approximately 150 genera and over 2,000 species of cyanobacteria [@vincent_2009_cyanobacteria]. Cyanobacteria are morphologically diverse, ranging from unicellular to multicellular. They can be filamentous, coccoid, trichal, and some can have differential cells such as heterocysts or akinetes. These are Gram-Negative bacteria originally described under the Botanical, and it wasn't until 1978 that scientists proposed systems to resolve the classification of cyanobacteria and place them under the bacteriological code [@walter_2017_ecogenomics]. Currently, there is a formal system that is a compromise between the Bacteriological and Botanical Codes for species and is based on botanical taxonomic criteria and includes bacteriological and molecular information [@palinska_2014_taxonomy].  Their morphological diversity provided a challenge for scientists regarding classification. Still, their biochemical and physiological properties allowed them to live almost anywhere in extreme environments such as hypersaline lakes and calderas on active volcanoes. 

\

**1.2 Microbial Diversity and Housekeeping genes**

\

Locations such as the Hawaiian Archipelago provide a unique biodiversity hotspot due to the geographical isolation of these islands and the tropical climates year around. Well documented studies on microorganism communities and interactions are limited and there are still locations in which there has been little anthropogenic activity. Such locations provide ideally pristine habitats for the study of microbial diversity and potentially discovery of novel species. Locations such as underwater hydrothermal vents and lava calderas provide unique niches for organisms to thrive. 

However, to thrive in such environments, these organisms must have adapted a way to obtain energy. With the rise of cyanobacteria (blue-green algae), Chlorophyta (green algae), Rhodophyta (red algae), Bacillariophyta (diatoms), and Plantae (photosynthetic eukaryotes) can also be broken down into two types of photophosphorylation: oxygenic phosphorylation and anoxygenic phosphorylation. Cyanobacteria and Eukaryotes fall under oxygenic phosphorylation, and produce O2 from H2O using chlorophyll as a light harvester and are considered photoautotrophs since they utilize light as energy and will fix CO2. 

The process of gaining energy is a cycle called the Calvin-Benson-Bassham (CBB). This process is a reductive pentose phosphate pathway that assimilates CO2. Ribulose- 1,5-bisphosphate carboxylase/oxygenase (RubisCO) is one of the major enzymatic activities unique to the Calvin cycle and is also the CBB enzyme responsible for the fixation for CO2 [@hirota_2002_isolation]. These genes encoding RubisCO are broken down into Ribulose bisphosphate carboxylase large chain (cbbL) and Ribulose bisphosphate carboxylase small chain (cbbS). While the compromise between the Botanical and Bacteriological Code for new species includes, phylogenetic analyses investigate the sequence divergence of the 16s rRNA gene, construction of the secondary structure of the 16s - 23s region, morphological description, and description of the thylakoid structure, recently papers like [@mare_2019_taxonomic],  [@xie_1989_cyanobacterial], and [@walter_2017_ecogenomics] have been published using additional housekeeping genes to help resolve placement of species within a Genera. 

Gene transcription is directed through the DNA-dependent RNA polymerase, which in bacteria, is formed by a core RNA polymerase broken down into the sigma factors: α, β, β′, and σ.  The DNA-directed RNA polymerase subunit β (rpoB)and the DNA-directed RNA polymerase subunit β′ (rpoC and rpoC2) are adjacent to each other and are transcribed from the major promoter (Xie, Wen - Qin et al. 1989 ). Due to the 16s rRNA region lacking sensitivity for evolutionary changes, these additional housekeeping genes are being added to studies more recently. 

Novel species of cyanobacteria were sampled from a volcano caldera located on Kiluea, HI in 2008. Three species of cyanobacteria have been isolated from a consortium of organisms growing in a mat on the cave wall.  We sequenced the cyanobacteria, all three of which were novel species, so following the compromise between the Botanical and Bacteriological Code, we will describe the morphology of the organisms, construct a secondary structure of the 16s - 23s region, and do a phylogenetic analysis of the whole genomes, rpoB gene, rpoC gene, rpoC2 gene, cbbL gene, and cbbS gene. We would also like to investigate the production of antibiotics and inhibitory genes of these novel species.

\

## Description of data and data source

*Novel species was discovered from Big Ell Cave located on Kilauea, Hawaii in 2008. Mutliple samples were taken from 2008 - 2018 in order to get an idea of the effects of a volcanic eruption on the biodiversity within the caldera. Samples were isolated to obtain what we are calling JS1 (Gleobacter violaceus), JS2 (Fischerella sp), and JS3 (Leptolygnbya sp). GitHub repository (https://github.com/Yokohime/Chiyoko_ZOOL710_project3.git) has the raw data, coding scripts for cleaning and analysis.*

![Figure.1: Location of caldera located on Kiluea, HI.](Supplement/Kileaua_location.jpg)

![Figure.2: Image of mat sampled from within Big Ell cave containing JS1 (purple) , JS2 (green) , JS3 (green), and including a multitude of other bacteria.](Supplement/Big_Ell_Mat.JPG)

## Questions/Hypotheses to be addressed

*Aims of this project include to address the following:*

-   <div>

    1.  Are these unknown cyanobacteria novel Species, Genera, or Family?

    </div>

-   <div>

    2.  Do these novel cyanobacteria--Leptolyngbya sp. and Fischerella sp.--produce any toxic secondary metabolites or any that inhibit the survival of another organism?

    </div>

-   <div>

    3.  If these species produce any toxic or inhibitory secondary metabolites, are they novel?

    </div>

{{< pagebreak >}}

# Methods

After extracting the DNA, Illumina and MinION sequencing was done in the Advanced Studies in Genomics, Proteomics, and Bioinformatics (ASGPB) located at the University of Hawai'i at Manoa, Life Science Building 320. Genomic data for all available Fischerella species were obtained from National Center for Biotechnologu Information (NCBI). Genomes were annotated using Ulana, a Prokka pipeline written by Ethan Hill of Iolani High School. Using Rstudio, the annotations were cleaned and organized to extract the annotated genes of interest. Genes for the rpoB, rpoC, rpoC2,cbbL, and cbbS genes were extracted from the genome and aligned using MUSCLE 3.8 which can be accessed on European Molecular Biology Laboratory (EMBL-EBI):https://www.ebi.ac.uk/Tools/msa/.

## Data acquisition, import and cleaning

\*Data was obtained from field samples of a mat found in a volcanic calderea on the island, Hawai'i in 2008. DNA was extracted and sequenced at the facilities located at the University of Hawai'i at Manoa. The whole genome was annotated and cleaned and can be found under under the Raw_data folder.

**Cleaning script**

*Data was uploaded, checked for errors, and then organized to extract the data of interest. Comparison between each species was made in terms of what genes each species shared between each other as well as what genes were shared between all species. Cleaning script can be found here: https://github.com/Yokohime/Chiyoko_ZOOL710_project3/blob/main/Code/Processing_Code/ProcessingData.R*

{{< pagebreak >}}

# Results

## Exploratory/Descriptive analysis

# 16s

```{r}
#| label: fig.3:Phylogenetic tree based on the 16S rRNA gene.
#| fig-cap: "Maximum-likelihood tree based on the 16S gene sequences of all available Fischerella species. Bootstrap values after 1,000 replicates are expressed as percentages.Gleobacter violaceus PCC 7421 was used as an outgroup. The scale bar indicates the fraction of nucleotide substitutions per site."
#|echo: FALSE
 knitr::include_graphics("Images/16S_Tree.png")

```

```{r}
#| label: fig.4:Patrisitic distances for the 16S rRNA gene.
#| fig-cap: "The sume of the branch lengths for the 16S gene for all 7 species of Ficherella sp. and Gleobactor vialoceus."
#|echo: FALSE
 knitr::include_graphics("Images/16S_Matrix.png")
```

# rpoB

```{r}
#| label: fig.5:Phylogenetic tree based on the rpoB gene.
#| fig-cap: "Maximum-likelihood tree based on the rpoB gene sequences of all available Fischerella species. Bootstrap values after 1,000 replicates are expressed as percentages.Gleobacter violaceus PCC 7421 was used as an outgroup. The scale bar indicates the fraction of nucleotide substitutions per site."
#|echo: FALSE
knitr::include_graphics("Images/rpoB_Tree.png")

```

```{r}
#| label: fig.6:Patrisitic distances for the rpoB gene.
#| fig-cap: "The sume of the branch lengths for the rpoB gene for all 7 species of Ficherella sp. and Gleobactor vialoceus."
#| echo: FALSE
knitr::include_graphics("Images/rpoB_Matrix.png")

```

# rpoC

```{r}
#| label: fig.7:Phylogenetic tree based on the rpoC gene.
#| fig-cap: "Maximum-likelihood tree based on the rpoC gene sequences of all available Fischerella species. Bootstrap values after 1,000 replicates are expressed as percentages.Gleobacter violaceus PCC 7421 was used as an outgroup. The scale bar indicates the fraction of nucleotide substitutions per site."
#| echo: FALSE
knitr::include_graphics("Images/rpoC_Tree.png")

```

```{r}
#| label: fig.8:Patrisitic distances for the rpoC rRNA gene.
#| fig-cap: "The sume of the branch lengths for the rpoC gene for all 7 species of Ficherella sp. and Gleobactor vialoceus."
#|echo: FALSE
knitr::include_graphics("Images/rpoC_Matrix.png")
```

# rpoC2

```{r}
#| label: fig.9:Phylogenetic tree based on the rpoC2 gene.
#| fig-cap: "Maximum-likelihood tree based on the rpoC2 gene sequences of all available Fischerella species. Bootstrap values after 1,000 replicates are expressed as percentages.Gleobacter violaceus PCC 7421 was used as an outgroup. The scale bar indicates the fraction of nucleotide substitutions per site."
#| echo: FALSE
knitr::include_graphics("Images/rpoC2_Tree.png")
```

```{r}
#| label: fig.10:rpoC2 
#| fig-cap: "Percent Identity based on the rpoC2 gene for all 7 species of Ficherella sp. and Gleobactor vialoceus."
#|echo: FALSE
knitr::include_graphics("Images/rpoC2_Matrix.png")
```

# rpo Genes concatenated

```{r}
#| label: fig.11:Phylogenetic tree based on the rpoB, rpoC, and rpoC2 genes.
#| fig-cap: "Maximum-likelihood tree based on the rpo gene sequences of all available Fischerella species. Bootstrap values after 1,000 replicates are expressed as percentages.Gleobacter violaceus PCC 7421 was used as an outgroup. The scale bar indicates the fraction of nucleotide substitutions per site."
#|echo: FALSE
knitr::include_graphics("Images/rpo_Gene_Tree.png")
```

```{r}
#| label: fig 12:Patrisitic distances for the rpo genes.
#| fig-cap: "The sume of the branch lengths for the rpo genes for all 7 species of Ficherella sp. and Gleobactor vialoceus."
#|echo: FALSE
knitr::include_graphics("Images/rpo_Gene_Matrix.png")
```

# cbbL

```{r}
#| label: fig.13:Phylogenetic tree based on the cbbL gene.
#| fig-cap: "Maximum-likelihood tree based on the cbbL gene sequences of all available Fischerella species. Bootstrap values after 1,000 replicates are expressed as percentages.Gleobacter violaceus PCC 7421 was used as an outgroup. The scale bar indicates the fraction of nucleotide substitutions per site."
#| echo: FALSE
knitr::include_graphics("Images/cbbL_Tree.png")
```

```{r}
#| label: fig 14:Patrisitic distances for the cbbL gene.
#| fig-cap: "The sume of the branch lengths for the cbbL gene for all 7 species of Ficherella sp. and Gleobactor vialoceus."
#|echo: FALSE
knitr::include_graphics("Images/cbbL_Matrix.png")
```

# cbbS

```{r}
#| label: fig.15:Phylogenetic tree based on the cbbS gene.
#| fig-cap: "Maximum-likelihood tree based on the cbbS gene sequences of all available Fischerella species. Bootstrap values after 1,000 replicates are expressed as percentages.Gleobacter violaceus PCC 7421 was used as an outgroup. The scale bar indicates the fraction of nucleotide substitutions per site."
#| echo: FALSE
knitr::include_graphics("Images/cbbS_Tree.png")
```

```{r}
#| label: fig 16:Patrisitic distances for the cbbS gene.
#| fig-cap: "The sume of the branch lengths for the cbbS gene for all 7 species of Ficherella sp. and Gleobactor vialoceus."
#|echo: FALSE
knitr::include_graphics("Images/cbbS_Matrix.png")
```

# All genes combined

```{r}
#| label: fig.17:Phylogenetic tree based on the cbbL and cbbS genes.
#| fig-cap: "Maximum-likelihood tree based on the cbbLand cbbS gene sequences of all available Fischerella species. Bootstrap values after 1,000 replicates are expressed as percentages.Gleobacter violaceus PCC 7421 was used as an outgroup. The scale bar indicates the fraction of nucleotide substitutions per site."
#|echo: FALSE
knitr::include_graphics("Images/cbb_Tree.png")
```

```{r}
#| label: fig 18:Patrisitic distances for the cbb genes.
#| fig-cap: "The sume of the branch lengths for the cbb genes for all 7 species of Ficherella sp. and Gleobactor vialoceus."
#|echo: FALSE
knitr::include_graphics("Images/cbb_Matrix.png")
```

# All genes combined

```{r}
#| label: fig.19:Phylogenetic tree based on the 16S rRNA, rpoB, rpoC, rpoC2, cbbL,and cbbS genes.
#| fig-cap: "Maximum-likelihood tree based on the 16S rRNA, rpoB, rpoC, rpoC2, cbbL,and cbbS gene sequences of all available Fischerella species. Bootstrap values after 1,000 replicates are expressed as percentages.Gleobacter violaceus PCC 7421 was used as an outgroup. The scale bar indicates the fraction of nucleotide substitutions per site."
#| echo: FALSE
knitr::include_graphics("Images/All_Genes_Tree.png")

```

```{r}
#| label: fig 20:Patrisitic distances for the rpoB, rpoC, rpoC2, cbbL, cbbS, and 16S rRNA genes.
#| fig-cap: "The sume of the branch lengths for the rpoB, rpoC, rpoC2, cbbL, cbbS, and 16S rRNA genes for all 7 species of Ficherella sp. and Gleobactor vialoceus."
#|echo: FALSE
knitr::include_graphics("Images/All_Genes_Matrix.png")
```

# Products of JS2

```{r}
require(kableExtra)
#| label: tbl-Products of JS2
#| tbl-cap: "List of products produced based on the genes present in JS2."
#| echo: FALSE
options(knitr.kable.NA = "")
resulttable=readRDS("../../Data/Processed_Data/JS2Products.rds")
knitr::kable(resulttable, digits=1) %>%
  kable_styling()%>%
  scroll_box(width = "100%", height = "200px")
```

## Further analyses

The trees give a nice visual of relationship between the species while comparing their relationships based on genes of interest. With more time, it is necessary to investigate the secondary structure of the 16S - 23S region of these cyanobacteria to identify the hairpin loop structures. Furthermore, although products of JS2 were determined, further investigation of the products needs to be done to better understand the secondary metabolites produced by JS2. Similarly, once the products have been thoroughly investigated, the MSMS should be preformed to determine if there are any produced at the time of extraction.

{{< pagebreak >}}

# Discussion

## Summary and Interpretation

For this project, the cleaned data was taken and analyzed using Rstudio to make graphs and plots to help visualize the data. Three questions were posed at the beginning, 1. The individual genes did yield different phylogenetic trees, however, in the majority of trees we see a pattern of JS2 being grouped near F.thermalis, F.major, and F.muscicola, 2. There were toxic and inhibitory genes based on the genome annotations for JS2 and 3. Additional testing will need to be done to determine if these genes are being expressed at the time of extraction.

## Strengths and Limitations

This study is to investigate the presence of toxic, antibiotic, and/or inhibitory genes within the whole genome of our novel species. Thus, one huge factor to this experiment will be whether or not the secondary metabolite is being expressed when the MSMS and/or extraction is preformed. Secondary metabolites such as those I am interested are not consistently produced as this would be energetically expensive. Stressing the cyanobacteria out with environmental factors such as temperature, humidity, light, or pH will most likely have little to no effect on the secondary metabolites of interest, however introducing another bacterial species has its own problems. The current struggle is to keep these cultures anexic due to the fact that contaminates will add secondary metabolites not produced by the cyanobacteria, however, they can cause the cyanobacteria to produce secondary metabolites as a response mechanism. Another way to potentially obtain secondary metabolites of interest if they are not expressed, or in high enough concentrations would be to create a vector in another organism and promote the expression in that organism since the cyanobacteria grow so slow already.

## Conclusions

To summarize, we found that there is a difference between phylogentic genetic trees, and more specifically, there is a difference between the placement of species between the phylogentic tree based on all genes concatenated verses each individual genes.

{{< pagebreak >}}

# References
